docs/astro/sql/operators/load_file.rst

```diff
@@ -0,0 +1,272 @@
+======================================
+load_file operator
+======================================
+
+When to Use load_file operator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are two use cases of the load_file operator.
+
+#. When we want a file to be loaded into a database table
+#. When we want a file to be converted to dataframe
+
+Case 1: Load into a database table
+    you need to pass the `output_table` param to the load_file operator to convert it to a table and the operator returns an instance of the table object passed in `output_table`. The output table will be created if it doesn't exist and will be replaced if it does. We can change this behavior with the `if_exists` parameter.
```

Comment:
Should we cover the difference between creating a named Table vs. an unnamed one (temp table)?

Reply:
Great point, @mikeshwe! @utkarsharma2 is covering this in the concepts and cleanup documentation (https://github.com/astronomer/astro-sdk/pull/609). What do you think if we add here a reference to the Table concepts section?

Reply:
@mikeshwe I thought so too, but I thought it should be on a separate page. Please check the PR  - https://github.com/astronomer/astro-sdk/pull/609